compiler/AST/checkAST.cpp

```diff
@@ -214,8 +216,6 @@ void checkPrimitives()
      case PRIM_ARRAY_SHIFT_BASE_POINTER:
      case PRIM_ARRAY_SET:
      case PRIM_ARRAY_SET_FIRST:
-     case PRIM_ERROR:
-     case PRIM_WARNING:
```

Comment:
I'm not sure I follow this. `checkPrimitives()` loops over all CallExprs so I don't think it's possible for PRIM_WARNING to exist past resolution on master right now. I don't see how changing a later pass would affect what primitives are in the AST at resolution time.